README.md

```diff
@@ -10,15 +10,18 @@ to support buck2/ocaml for OSS.
 *Disclaimer: rules.py is hardcoded to my (vsiles) current setup, please update it
 to fit yours until we make a proper reusable script ;)*
```

Comment:
I'd say we can change this disclaimer. It at least works for me too ;)

Reply:
I think it makes sense, yes, with the changes you provide.